IDEAS/LIDEAS/Components/BaseClasses/LinearAirModel.mo

```diff
@@ -0,0 +1,87 @@
+within IDEAS.LIDEAS.Components.BaseClasses;
+model LinearAirModel
```

Comment:
I don't think we will still need a `LinearAirModel` since the existing `WellMixedAir` can be used directly?